### PR TITLE
got bored and decided to run codespell on the code

### PR DESCRIPTION
**Pull request by ocefpaf**
Pretty much what the title says.

### CODE REVIEW

**Comment by mwengren** [comment]
I can never remember how to spell 'occurred', thanks!

**Comment by ocefpaf** [comment]
> I can never remember how to spell 'occurred', thanks!

I'm experimenting with this as a GH action. It worked wonders in https://github.com/ioos/ioos-python-package-skeleton/pull/4/files

File: compliance_checker/tests/test_acdd.py [hunk 1]
```diff
@@ -424,7 +424,7 @@ def test_geospatial_bounds(self):
         empty_ds = Dataset(os.devnull, "w", diskless=True)
         self.addCleanup(empty_ds.close)
 
-        # Mispelled WKT. Error message should include the attribute checked
```
Comment (benjwadams) [inline compliance_checker/tests/test_acdd.py:427]: The irony...

**Comment by benjwadams** [comment]
Didn't know about this tool.  Will have to check it out for my Git hooks.

**Comment by ocefpaf** [comment]
> Didn't know about this tool. Will have to check it out for my Git hooks.

See https://github.com/ioos/compliance-checker/pull/825

**Pull request merged**